terraform/gce/gce.tf

```diff
@@ -130,3 +130,11 @@ resource "google_compute_instance" "mi-worker-nodes" {
 
   count = "${var.worker_count}"
 }
+
+output "control_ips" {
+  value = "${join(\",\", google_compute_instance.mi-control-nodes.*.access_config.nat_ip)}"
+}
```

Comment:
This should be fixed now. It's strange that it didn't work though, this is what the Terraform docs say: https://terraform.io/docs/providers/google/r/compute_instance.html#access_config